### ISSUE TITLE
Python interactive hangs

### ISSUE THREAD

**Issue by pschmied** [melpa/melpa#398]
I installed python prelude, then tried starting an interactive prompt. Emacs hangs taking 100% of one CPU. This is apparently a known issue, and has been fixed in some newer version of python.el, however the changes apparently haven't trickled down to the github repository from the (canonical?) repository repo.or.cz.

See this issue for the gory details.
https://github.com/fgallina/python.el/issues/117

In short, the hangs go away if I manually replace ~/.emacs.d/elpa/python-20121014.1458/python.el with file http://repo.or.cz/w/emacs.git/blob_plain/emacs-24:/lisp/progmodes/python.el

If the the repo.or.cz is actually the canonical repo, a fix for the python package might be to simply point to here?

Cheers,
Peter
